Replacing np.bool with np.bool_ (was deprecated since NumPy 1.20.0, removed in 1.24.0).
author: Franky Backeljauw (Universiteit Antwerpen)
--- Wrapping/Generators/Python/itk/support/types.py.orig	2025-06-11 16:36:33.000000000 +0200
+++ Wrapping/Generators/Python/itk/support/types.py	2025-08-14 19:57:04.751103441 +0200
@@ -140,7 +140,7 @@
         _SS: "itkCType" = itkCType("signed short", "SS", np.dtype(np.int16))
         _SI: "itkCType" = itkCType("signed int", "SI", np.dtype(np.int32))
         _SLL: "itkCType" = itkCType("signed long long", "SLL", np.dtype(np.int64))
-        _B: "itkCType" = itkCType("bool", "B", np.dtype(np.bool))
+        _B: "itkCType" = itkCType("bool", "B", np.dtype(np.bool_))
         return _F, _D, _UC, _US, _UI, _UL, _SL, _LD, _ULL, _SC, _SS, _SI, _SLL, _B
 
